### DISCUSSION TITLE
The error message always states "you cannot fire" even though you are reloaded

### DISCUSSION THREAD

**Discussion by mustangsami** [JDCalvert/FVTT-PF2e-Ranged-Combat#91] [General]
Hi there! Love this module. I'm using it in a game with a gunslinger and it's proving super useful, so thanks very much. I had a couple of ideas if you're open to them?

https://github.com/JDCalvert/FVTT-PF2e-Ranged-Combat/blob/a9e4b0c55141e6759e16908372ba5f02bbb67017/scripts/ammunition-system/actions/reload-magazine.js#L67

On this line for the error if a magazine cannot be added to a weapon because it's already loaded, states incorrectly that "you cannot fire this weapon", even though the macro used is reload magazine. 

I can see that your dialog box here always states this here:  https://github.com/JDCalvert/FVTT-PF2e-Ranged-Combat/blob/a9e4b0c55141e6759e16908372ba5f02bbb67017/scripts/utils/utils.js#L230

I also see that once you close off a dialog box with the "Do not show this again" label, it will never show ANY error again. Is there a way to re-enable it? Perhaps by a macro to add the compendium? They are super useful and rather than remove them entirely, I'd love to see them be a bit more subtle.

Kind regards,
Sami

**Comment by JDCalvert** [comment]
Hi there, thanks for your appreciation and feedback!

I actually had never realised that I'd allowed that popup to appear for times when you weren't trying to fire a weapon, I'll have to change the wording for that!

Regarding the "do not show this again" button, I could add another option in the module settings to re-enable the popup, but you should still get a small yellow warning banner in Foundry whenever the module stops you from doing something, for example trying to load a crossbow that's already loaded:

![image](https://user-images.githubusercontent.com/38069151/200509792-20c6b4fa-c15e-4994-b9ff-9b545db6b048.png)

Is that what you mean?

**Comment by mustangsami** [comment]
Hi Jonathan,

No worries. Was very glad to see a better way to handle reloading was
available.

Re the warning module, oh ok great! I was too afraid to click don't show
again to see that it happens, so that's ideal!

Thanks for letting me know.

I'll keep an eye out for any other potential improvement in the module.
Thanks again.

Sami
